### PR TITLE
Constrain basic channel to only owner

### PR DESCRIPTION
**Pull request by musnit**
This PR constrains the basic channel to an owner which should be the bootstrap user.

In test/ropsten, we can setup a proxy contract to allow anyone to still use it for testing/QA purposes.

### CODE REVIEW

File: ethereum/contracts/TestBasicInboundChannelProxy.sol [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./ParachainLightClient.sol";
+import "./BasicInboundChannel.sol";
+
+contract TestBasicInboundChannelProxy is Ownable {
```
Comment (philipstanislaus) [inline ethereum/contracts/TestBasicInboundChannelProxy.sol:8]: Could you add a comment describing what the idea of this contract is? Why do we test this case?

**Review by philipstanislaus — APPROVED** [review]

File: ethereum/contracts/TestBasicInboundChannelProxy.sol (hunk 1 above)
Comment (musnit) [reply ethereum/contracts/TestBasicInboundChannelProxy.sol]: renamed and commented

**Comment by musnit** [comment]
@vgeddes will wait for your review before merging on this one

**Comment by musnit** [comment]
@vgeddes going to merge this - plz review and share any comments if necessary after merge

**Pull request merged**